charts/tidb-cluster/values.yaml

```diff
@@ -396,6 +397,7 @@ tidb:
   service:
     type: NodePort
     exposeStatus: true
+    # portName: "<the default name will be used>"
```

Comment:
```suggestion
      portName: mysql
```

Reply:
If you do this, there will be a problem of backward compatibility. When upgrading the operator, trigger the upgrade of the cluster.